homeworks/HW1/src/Courses.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Button, Form, FormGroup, Input, Col, Table, Row } from 'reactstrap';
+import { Link } from 'react-router-dom';
+import './App.css'
+export default class Courses extends React.Component {
+
+    renderCourses = () => {
+        const courses = [{
+            title: "React Redux",
+            duration: "16h",
+            description: `Adipisicing elit.
+            Harum pariatur porro quam est exercitationem rem ut corrupti
+            tempore! Mollitia ad similique dolore voluptas.
+            Corrupti saepe placeat blanditiis aliquam totam sed?`
+        },
+        {
+            title: "Node.js",
+            duration: "36h",
+            description: `Lorem ipsum dolor sit amet consectetur, adipisicing elit.
+            Nemo ratione enim laudantium aspernatur, cupiditate quis
+            placeat sapiente minus dolore ab necessitatibus aliquam magni
+            earum neque assumenda accusantium veniam est voluptatibus`
+        },
+        {
+            title: "Ajax",
+            duration: "6h",
+            description: `Nemo ratione enim laudantium aspernatur, cupiditate quis
+            placeat sapiente minus dolore ab necessitatibus aliquam magni
+            earum neque assumenda accusantium veniam est`
+        }];
+        console.log(courses)
+        return courses.map((elem, index) => {
```

Comment:
Would be better to name `elem` as a `course`.

Reply:
changed